docker_and_oci.rst

```diff
@@ -519,6 +531,21 @@ header. The following two examples are equivalent:
    Registry: quay.io
    From: bitnami/python:3.7
 
+Build architecture
+------------------
+
+The ``--arch`` option accepts a CPU architecture only.
+For example, to build an image for a 64-bit ARM system:
+
+.. code::
+
+   $ {command} build --arch arm64 ubuntu.sif docker://ubuntu
+
+The ``--arch-variant`` option can be used for CPU variants.
+Used for fetching the image, but not recorded in the build.
```

Comment:
I don't understand what you're trying to say in this line.  What does "not recorded in the build" mean?

Reply:
In the build metadata, it will be recorded using the arch **excluding** the variant (GOARCH).

Not sure if that is a SIF standard thing, or just an implementation detail (not using GOARM)?

Reply:
I see.  Then I suggest
```
The ``--arch-variant`` option can be used for fetching CPU variants,
although the variant is not recorded in the SIF file.
```